articles/proofs-arguments-and-zero-knowledge/chapter_19.md

```diff
@@ -0,0 +1,174 @@
+| IPs                            | FRI-based (multilinear) polynomial commitments                                       | Virgo                                            |
+| IPs                            | Discrete-log-based (multilinear) polynomial commitments (Bulletproofs, Hyrax-commit) | Hyrax                                            |
+| IPs                            | KZG-based (multilinear) polynomial commitments                                       | zk-vSQL and Libra                                |
+| MIPs                           | Multilinear polynomial commitments                                                   | Spartan, Xiphos, Kopis, Brakedown, and Shockwave |
+| Constant-round polynomial IOPs | FRI-based (univariate) polynomial commitments                                        | Aurora, Fractal, and Redshift                    |
+| Constant-round polynomial IOPs | KZG-based (univariate) polynomial commitments                                        | Marlin                                           |
+| Constant-round polynomial IOP  | Ligero                                                                               | Ligero                                           |
+
+Beside that, the most popular variant of the SNARK derived from GGPR’s linear PCP is
+Groth16.
+
+**More SNARKs via composition**: On top of the taxonomy of SNARKs delineated above, one can take any two SNARKs designed via one of the above approaches, and compose them one or more times. Such compositions are growing increasingly popular and already yield state-of-the-art performance.
+
+## 19.2 Pros and Cons of the Approaches
+
+**Pros and cons of three first approach in section 10.6**.
+
+a fifth approach to argument design, based on commit-and-prove techniques (13.1) (not so related)
+
+(IP-based, MIP-based, and constant-round-polynomial-IOP-based)
+
+the information-theoretically secure protocol
+ +
+ any extractable polynomial commitment scheme
+ =
+
+  succinct argument
+
+Only one technique to to turn linear PCPs into publicly-verifiable SNARKs, based on pairings and very similar to KZG polynomial commitments,
+
+IP-based and MIP-based argument systems, the polynomial commitment scheme must allow committing to multilinear polynomials.
+
+the IOP-based argument system, the polynomial commitment scheme must allow committing to univariate polynomials
+
+three broad **approaches** to polynomial commitment schemes
+ some of these approaches have multiple instantiations with various cost tradeoff
+
+ IOPs combined with Merkle hashing, where we saw FRI
+
+ transparent Σ-protocols that assume hardness of the discrete logarithm problem, where we saw Hyrax-commit (Section 14.3), Bulletproofs (Section 14.4), and Dory
+
+ KZG [KZG10] and uses pairings and a trusted setup (Section 15.2)
+
+    = 
```

Comment:
what equal what?

Reply:
Hmm it is a MrSon's writing. So I think I need to modify it with him